packages/web/components/tokens/stitches.config.ts

```diff
@@ -203,90 +203,21 @@ export const darkTheme = createTheme(ThemeId.Dark, {
     readerHeader: '#b9b9b9',
     readerTableHeader: '#FFFFFF',
   },
-})
-
-export const darkerTheme = createTheme(ThemeId.Darker, {
-  colors: {
-    ...yellowDark, // Brand
-    ...orangeDark, //Accent
-
-    // Grayscale
-    grayBase: grayDark.gray1,
-    grayBgSubtle: grayDark.gray2,
-    grayBg: grayDark.gray3,
-    grayBgHover: grayDark.gray4,
-    grayBgActive: grayDark.gray5,
-    grayLine: grayDark.gray6,
-    grayBorder: grayDark.gray7,
-    grayBorderHover: grayDark.gray8,
-    graySolid: grayDark.gray9,
-    graySolidHover: grayDark.gray10,
-    grayText: grayDark.gray11,
-    grayTextContrast: grayDark.gray12,
-
-    // Semantic Colors
-    overlay: blackA.blackA9,
-    highlightBackground: 'rgb(255,234,159)',
-    highlight: '#FFD234',
-    highlightText: 'rgba(255, 210, 52)',
-    error: '#FA5E4A',
-
-    // Brand Colors
-    loginBg: '#FFD492',
-
-    // Reader Colors
-    readerBg: '#000000',
-    readerFont: '#888888',
-    readerFontTransparent: 'rgba(136,136,136,0.65)',
-    readerHeader: '#888888',
-    readerTableHeader: '',
-  },
-})
-
-export const lighterTheme = createTheme(ThemeId.Lighter, {
-  colors: {
-    ...yellow, // Brand
-    ...orange, //Accent
-
-    // Grayscale
-    grayBase: '#F8F8F8',
-    grayBgActive: '#FFFFFF',
-    grayBorder: 'rgba(0, 0, 0, 0.06)',
-    grayTextContrast: '#3A3939',
-
-    // Grayscale
-    grayBgSubtle: gray.gray2,
-    grayBg: gray.gray3,
-    grayBgHover: gray.gray4,
-    grayLine: gray.gray6,
-    grayBorderHover: gray.gray8,
-    graySolid: gray.gray9,
-    graySolidHover: gray.gray10,
-    grayText: gray.gray11,
-
-    // Semantic Colors
-    overlay: blackA.blackA9,
-    highlightBackground: 'rgb(255,234,159)',
-    highlight: '#FFD234',
-    error: '#FA5E4A',
+}
 
-    // Brand Colors
-    loginBg: '#FFD492',
+// Dark and Darker theme now match each other.
+// Use the darkThemeSpec object to make updates.
+export const darkTheme = createTheme(ThemeId.Dark, darkThemeSpec)
```

Comment:
We might be able to remove these as well. As long as nobody has preferences for lighter and darker in the backend (or we convert on the client...which we might do. will check)

Reply:
(meant to refer to darker and lighter themes)

Reply:
Yeah hopefully we convert, I'd be worried about removing because it could be some of the oldest users that would have them.

Reply:
Probably better to leave these in for now. I can't tell if we're converting it. Could do a follow up PR to clean it up but at least this one gets rid of the extra noise and makes it easier to see where updates are needed when we add to the theme objects.